packages/jest-cli/src/TestRunner.js

```diff
@@ -115,6 +115,11 @@ class TestRunner {
     reporter.onRunStart && reporter.onRunStart(config, aggregatedResults);
 
     const onTestResult = (testPath, testResult) => {
+      if (testResult.testResults.length === 0) {
+        const EmptySuiteError = require('jest-util').EmptySuiteError;
```

Comment:
this should be required on the top. We also usually sort requires by alphabet up there :) (with one block for capitalized modules and one for lowercased modules).